plugins/nodes/src/js/data/NodesNetworkClient.ts

```diff
@@ -11,6 +11,15 @@ export interface NodeNetwork {
 
 export type NodesNetwork = NodeNetwork[];
 
+export const NodeNetworkSchema = `
+type NodeNetwork {
+  updated: String
```

Comment:
Isn't this a Date?

Reply:
It is an ISO datetime string.

Reply:
Do we check somewhere if it's the correct format and not just any string?